netsim/extra/bgp.session/dellos10.j2

```diff
@@ -0,0 +1,47 @@
+{% macro ebgp_neighbor(n,af) -%}
+{%   set peer = n[af] if n[af] is string else 'interface ' + n.local_if|default('?') %}
+  neighbor {{ peer }}
+{%   if n.bfd|default(False) %}
+   bfd
```

Comment:
IIRC BFD has to be enabled also globally and on all the interfaces where the bfd session will be established.

Maybe you forgot to commit the bfd module template for os10? (I don't find it in the repo)

Reply:
Yep, forgot to commit the new file. Added now